.env.sample

```diff
@@ -1,6 +1,7 @@
 INTERVAL=20
 TIMEZONE=America/Sao_Paulo
 
+POST_TWITTER=False
```

Comment:
I rather avoid adding one more setting to this tiny app that already requires tons of settings – follow up in my comment on the `settings.py` file.

Reply:
Perfect! I'll fix that!